bot/cogs/match.py

```diff
@@ -224,6 +224,7 @@ class MatchCog(commands.Cog):
     def __init__(self, bot):
         """ Set attributes. """
         self.bot = bot
+        self.pending_task = None
```

Comment:
@thboss Actually I made a mistake here. This would be shared by all the Discord servers using the bot, and we don't want different servers sharing the same ready message variable. I know in your case that doesn't matter, but I am trying to keep the bot able to function across multiple servers. Could you use a dictionary here? Where the key is the `Guild` object, and the value is that server's `pending_task`. That way each server will have it's own `pending_task` and we can get it using the `Guild` object.